sbtc/src/rpc.rs

```diff
@@ -56,7 +58,7 @@ pub trait BitcoinClient {
 
     /// Return the transaction if the transaction is in the mempool or in
     /// any block.
-    fn get_tx(&self, txid: &Txid) -> Result<GetTxResponse, Self::Error>;
+    fn get_tx(&self, txid: &Txid) -> impl Future<Output = Result<GetTxResponse, Self::Error>>;
```

Comment:
Yeah I had it going the other way, but decided to change the trait here since right now the only user is `signer` and there we're async (and assuming emily etc. will be too).

Reply:
I see. 

Just a heads up, I don't think Emily will actually use this trait or any functionality where we fetch information from bitcoin-core (I originally thought that it would). So we should probably just move this code (and some other stuff) over to the `signer` crate.